packages/cms-lib/package.json

```diff
@@ -13,12 +13,14 @@
     "chokidar": "^3.0.1",
     "content-disposition": "^0.5.3",
     "debounce": "^1.2.0",
+    "express": "4.16.3",
```

Comment:
One of the reasons that I am not fond of bringing `@hubspot/api-auth-lib` into `@hubspot/cms-lib` is that includes express. This makes using `@hubspot/cms-lib` in non-traditional ways inside of the context of a web app that is built on top of express. It would be preferrer if somehow the authentication flow that uses `open` and `express` were part of the `@hubspot/cms-cli` and the logic needed to make API requests using OAuth and refreshing token lived in `@hubspot/cms-lib`. Does that make sense?